src/main/java/nvt/kts/ticketapp/controller/event/EventController.java

```diff
@@ -51,4 +54,22 @@ private ResponseEntity save (@RequestBody EventEventDaysDTO eventEventDaysDTO){
 
          return new ResponseEntity<Page<Event>>(eventService.findAll(pageable),HttpStatus.OK);
      }
+
+    @PutMapping("/{id}")
+    public ResponseEntity<EventDTO> update(@PathVariable(value = "id") Long eventId,
+                                           @RequestBody EventDTO eventDetails){
+
+        eventService.update(eventId,eventDetails);
```

Comment:
iz servisa vrati objekat koji je promenjen i njega vrati kao kao DTO u responsu. Znas zasto to.. tebi u ovom slucaju bice isto.. ali nekad kad menjas objekat mozda ces ti promeniti neke atibute explicitno pa sa ovim pristupom to neces resiti..

Reply:
Reseno